reference/orders.v2.oas2.yml

```diff
@@ -5058,6 +5077,31 @@ components:
               type: array
               items:
                 $ref: '#/components/schemas/giftCertificateConsignment_Get'
+    giftCertificateConsignment_Post:
+      x-examples:
+        example-1:
+          recipient_email: janedoe@example.com
+          line_items:
+            - gift_certificate_id: 11
+      type: object
+      properties:
+        recipient_email:
+          description: ''
+          example: 'janedoe@example.com'
+          type: string
+        line_items:
+          type: array
+          minItems: 1
+          items:
+            type: object
+            properties:
+              gift_certificate_id:
+                description: ''
+                example: 2
+                type: integer
```

Comment:
maybe this field should be marked as "required" as well

Reply:
https://github.com/bigcommerce/api-specs/pull/1013/commits/583247189e29a248b7e000b54902008ae16e584c